assets/asset_makers/ptable/ptable_matplotlib/ptable_heatmap_splits_plotly.py

```diff
@@ -0,0 +1,26 @@
+# %%
+import numpy as np
+from pymatgen.core.periodic_table import Element
+
+import pymatviz as pmv
+
+
+# %% Evenly-split tile plots laid out as a periodic table
+rng = np.random.default_rng(seed=0)
+for n_splits in (2, 3, 4):
+    data_dict = {
+        elem.symbol: rng.integers(10 * n_splits, 20 * (n_splits + 1), size=n_splits)
+        for elem in Element
+    }
+
+    fig = pmv.ptable_heatmap_splits_plotly(
```

Comment:
Looks like a plotly example sneaked into matplotlib example

Reply:
oops, that's by accident. let's revert

Reply:
Looks like it's the same as the `ptable_heatmap_splits_plotly` under `ptable_plotly`, so I would remove it 373bce31bb5d0859b1e309e2385aef9efa0400d4 instead of relocating.

Reply:
no i changed that from `ptable_heatmap_splits` to `ptable_heatmap_splits_plotly` by accident in #246. you originally wrote this code and i used it to check how different the output `ptable_heatmap_splits` and `ptable_heatmap_splits_plotly` looks but forgot to revert the temp change